### DISCUSSION TITLE
Webman Fan ramp up for physical PS2 media?

### DISCUSSION THREAD

**Discussion by Brandicus** [aldostools/webMAN-MOD#559] [Q&A] [answered]
Hi, first big thank you to Aldo and other creators for making modding on PS3 possible. I have had so many countless hours of enjoyment thanks to people like you. 

I have done some googling and it looks like you have answered this before, but im not sure what the solution is. ISOs loaded from webman ramp up to the expected 40%, physical DVD media does not. The very first time i put a PS2 DVD in my A01 it performed as expected but has not ramped up since. Can you please assist me with restoring this functionality?

A01 running Evilnat 4.88 and the latest version of Webman MOD from brewology. Id rather not have the fan running at 40% all of the time as its just been delidded and doesnt generally need to run anywhere close to that.

**Comment by aldostools** [comment, marked as answer]
Thank you for your message. webMAN MOD only is aware of the games that are mounted through its menu.
That means that physicals PS2 discs cannot be detected when they are launched.

Evilnat's 4.88.2 has an option to set fan settings dynamically when webMAN MOD is not running.
I'm not sure if that option works during PS2 game play, but it would worth to try.

A workaround would be to dump you PS2 games to ISO. It would activate the faster fan when mounted through webMAN MOD. Also your disc drive laser would last longer.

**Reply by Brandicus** [reply]
Thank you Aldo! I had seen you correspond with someone a couple years ago and it appeared as if you were working with them on a solution. Since that is not the case I will see if I can come up with a solution using evilnat settings. Thank you for all of your hard work, I appreciate you!

**Reply by Brandicus** [reply]
Aldo and anyone else who reads this. Aldo was correct there is a great workaround using the built in Evilnat fan control!

I observed the expected fan ramp up by setting the PS2 FAN setting in Evilnat COBRA settings menu. 

Please note that the behavior is different from Webman. Webman will ramp up as soon as the disc is mounted, but the disc workaround will only ramp up once the disc is launched. This is likely when webman stops controlling fan as Aldo stated.
